requirements.txt

```diff
@@ -1,2 +1,3 @@
 mxnet==0.10.0
 numpy>=1.12
+tensorboard>=1.0.0a3
```

Comment:
that sounds like the better option to me. The callback unit test also does not use tensorboard.